borg/archiver.py

```diff
@@ -112,7 +113,7 @@ def print_warning(self, msg, *args):
 
     def print_file_status(self, status, path):
         if self.output_list and (self.output_filter is None or status in self.output_filter):
-            logger.info("%1s %s", status, remove_surrogates(path))
+            logger.getLogger('borg.output.list').info("%1s %s", status, remove_surrogates(path))
```

Comment:
I dropped the logger convenience method and just use logging.getLogger() directly in the latest commit.